.circleci/config.yml

```
@@ -10,54 +12,34 @@ defaults: &defaults
 # restore_repo: &restore_repo
 #   restore_cache:
 #     key: v1-repo-{{ .Branch }}-{{ .Revision }}
-install_js: &install_js
-  run:
-    name: Install js dependencies
-    command: |
-      yarn config set yarn-offline-mirror ~/.cache/npm-packages-offline-cache/v1
-      yarn
-restore_yarn_cache: &restore_yarn_cache
-  restore_cache:
-    key: v2-yarn-sha-{{ checksum "yarn.lock" }}
-restore_yarn_offline_mirror: &restore_yarn_offline_mirror
-  restore_cache:
-    key: v2-npm-packages-offline-cache
-version: 2
+
+commands:
+  install_js:
+    steps:
+      - restore_cache:
+          key: v2-yarn-sha-{{ checksum "yarn.lock" }}
```

```suggestion
          keys:
             - v2-yarn-sha-{{ checksum "yarn.lock" }}
             - v2-yarn-sha-
```
fallback to using the latest cache if no exact match is found

---

Need to check again what the yarn cache actually does. It's not completely obvious to me that we would be ok here with a cache that was generated from a completely different lockfile.

---

Looks good to me. CircleCI docs are also using this fallback for caches depending on lockfiles: https://circleci.com/docs/2.0/caching/#restoring-cache

@Skaronator Thanks for the suggestion.